### DISCUSSION TITLE
disconnect relation if exists

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6037] [Q&A]
how to disconnect relation if exists whithout getting this error:
The records for relation `ProfileToUser` between the `User` and `Profile` models are not connected.

```ts
const user = await prisma.user.update({
  where: { email: 'bob  @prisma.io' },
  data: {
    profile: {
      disconnect: true,
    },
  },
})
```

**Comment by ryands17** [comment]
@BishoyNarouz 👋 

Currently Prisma throws an error if the relation doesn't exist and it cannot be configured for now. There's a feature request [here](https://github.com/prisma/prisma/issues/3069) for the same and it would be great if you could have a 👍 to this.

As a workaround, you would need to use a `try/catch` block around the call so that it doesn't throw an error.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
